packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts

```diff
@@ -74,7 +74,8 @@ export const topicNameLabelStyle = mergeStyles({
  */
 export const participantListWrapper = mergeStyles({
   boxShadow: '0px 0.3px 0.9px rgba(0, 0, 0, 0.1), 0px 1.6px 3.6px rgba(0, 0, 0, 0.13)',
-  width: '15rem',
+  width: '20rem',
```

Comment:
Why the bump to 20rem?

Reply:
children are 20rem, 15rem was truncating the pane and horizontal scrollbar was always there